generate.py

```diff
@@ -40,7 +40,7 @@
 from openedx_tagging.core.tagging.import_export import api as import_api
 from openedx.core.djangoapps.content_tagging.api import (
     create_taxonomy, get_taxonomies_for_org,
-    set_taxonomy_orgs, tag_content_object, get_content_tags,
+    set_taxonomy_orgs, tag_object, get_object_tags,
```

Comment:
Hm.. I'm not sure @pomegranited.
We re-export these methods in the `edx-platform` API.
I think we are using the methods from `edx-platform` instead of `oel_tagging` in all places (besides the views we don't override).

We don't need to change it right now, but what should we do for better consistency? Should we stop re-exporting and use `oel_tagging`, or use `edx-platform` and accept that the view will have a different path?

Reply:
I'm not fussed either way -- it doesn't matter enough to spend time on here :)

Reply:
Thank you @pomegranited!